### ISSUE TITLE
Add a help page to the Thimble website.

### ISSUE THREAD

**Issue by flukeout** [mozilla/thimble.mozilla.org#1647]
It would be good to have an easy to access help page somewhere on the site (or while editing). We could address some common issues on that page and maybe save people time. 

Recently we've been getting lots of emails about not being able to access the Your Projects page. It's resolved easily by a cache clear, but we don't have that information listed anywhere.

**Comment by humphd** [comment]
I made a start at this, but it's not useful being hidden in the wiki:

https://github.com/mozilla/thimble.mozilla.org/wiki/Using-Thimble

**Comment by flukeout** [comment]
We also DO link to the Sumo page from the header, so maybe we can just take advantage of that area. I think in particular what I think would be useful is to have a collection of steps that people can take to get unstuck. The stuff we usually recommend in email support exchanges:

* Are you in private browsing mode
* Clear your cache
* Clear your indexDB etc.

Additionally, when something goes wrong, we show the spinner indefinitely (or the black screen of doom) which obscures the link to the help page. We should probably then...

* Create a page on Sumo that helps people with the common fixes
* Add a link to the green loading page and black error page to this content

**Comment by flukeout** [comment]
Filed one of these issues in #1665

**Comment by jgmac1106** [comment]
I also have some pretty extensive tutorials I made but @humphd wiki is
pretty good. Why can't the help tab point to that instead of:
https://support.mozilla.org/en-US/products/webmaker/thimble which is so
utterly dated its like three logos ago.

**Comment by hannahkane** [comment]
There are advantages to updating the Sumo page - primarily that those questions and answers get localization support. I'd vote for updating those questions with newer content.

**Comment by jgmac1106** [comment]
That makes sense. I would love to take on the update if that is possible,
basically I would just remix the wiki into something that could end up on
Sumo for localisation.

If it doesn't make sense for me to do please let me know.

**Comment by hannahkane** [comment]
That would be great, @jgmac1106. Thanks! Just FYI, I believe content on Sumo needs to be in Question/Answer format.

Once you have something, we can all take a look, and then I can update Sumo whenever we're ready.

**Comment by jgmac1106** [comment]
Are there any other rules in terms of image size? The wiki and many of the tutorials use gif files.

**Comment by jgmac1106** [comment]
@hannahkane I was looking for the template files for SUMO and all the documentation I could find said Kitsune and sumo had reached EOL and were no longer being maintained. 

Is there a new place where SUMO lives or is the platform retired?

**Comment by hannahkane** [comment]
My understanding is that it's moving, but we can still be working on the content in the meantime.

**Comment by hannahkane** [comment]
See #1666

**Comment by Lakshmikanth-mkce** [comment]
@gideonthomas @flukeout I would like to work for this issue.

**Comment by jgmac1106** [comment]
@Lakshmikanth-mkce Thanks for looking to volunteer on the help page. Currently we redid the Q+A items. 

This issue might be on hold for a bit as we are waiting on some blockers and want to get a scope of what a help page could and should include.

We maybe organizing contributors in near future once this initial work is done and I can ping you then if still interested.
